### PR TITLE
404 newhomepage

### PR DESCRIPTION
**Pull request by kathleenahall**

### CODE REVIEW

**Review by JoshuaVSherman — CHANGES_REQUESTED** [review]
@kathleenahall there are issues with various screens sizes and cellphone display to fix please

**Comment by JoshuaVSherman** [comment]
@kathleenahall is this pull request any good still, i see you now have a new pull request with a different branch?

**Comment by kathleenahall** [comment]
I wasn't sure if I should make a new branch. I can merge them

**Comment by JoshuaVSherman** [comment]
@kathleenahall is this ready for me to review/test locally now?

File: test/containers/Homepage/Widescreen/index.spec.tsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
 describe('Homepage', () => {
   it('renders Homepage', () => {
-    const homepage = renderer.create(<Homepage />).toJSON();
-    expect(homepage).toMatchSnapshot();
-  });
-  it('renders WideAboutUs', () => {
-    const wideOrNarrow:any = renderer.create(<WideOrNarrow width={1005} />).toJSON();
-    expect(wideOrNarrow.props.className.includes('wideHome')).toBe(true);
+  //   const homepage = renderer.create(<NewHomepage />).toJSON();
```
Comment (JoshuaVSherman) [inline test/containers/Homepage/Widescreen/index.spec.tsx:6]: please delete commented out code in this file

**Review by JoshuaVSherman — CHANGES_REQUESTED** [review]

**Review by JoshuaVSherman — APPROVED** [review]

**Pull request merged**